doc/ctest_user_manual.md

```diff
@@ -305,3 +305,35 @@ The `CTest` harness supports out of the box string formatting and comparers for
 - `double`
 - `long double`
 - `char*` (`char_ptr`)
+
+### CTEST_DEFINE_ENUM_TYPE
+
+`CTEST_DEFINE_ENUM_TYPE` can be used with and enum defined by using the macro utils feature `MU_DEFINE_ENUM` in order to generate the needed `_ToString` and `_Compare` functions.
```

Comment:
>and [](start = 42, length = 3)

"an", here and below

Reply:
Done.

---
In reply to: [326827510](https://github.com/Azure/azure-ctest/pull/44#discussion_r326827510) [](ancestors = 326827510)